test_crates/enum_now_doc_hidden/new/src/lib.rs

```diff
@@ -0,0 +1,83 @@
+mod MyNonPublicMod {
+    // despite adding #[doc(hidden)], this enum is in a
+    // private mod, so it isn't part of the crate's public
+    // api
+    #[doc(hidden)]
+    pub enum MyEnum {
+        A,
+    }
+}
+
+pub mod MyPublicMod {
+    // added #[doc(hidden)], however this enum is in a
+    // public mod, so it is part of the crate's public api
+    #[doc(hidden)]
+    pub enum MyEnum {
+        A,
+    }
+}
+
+mod MyNestedNonPublicMod {
+    pub mod PublicInnerStruct {
+        // despite adding #[doc(hidden)], this enum is in a
+        // private outer mod, so it isn't part of the crate's public
+        // api
+        #[doc(hidden)]
+        pub enum MyEnum {
+            A,
+        }
+    }
+}
+
+pub mod MyNestedPublicMod {
+    pub mod PublicInnerStruct {
```

Comment:
copy-pasta?

Reply:
it's actually like this on the other test too, the name just plain doesn't make sense :/